File: app/src/main/java/huce/fit/appreadstories/dialog/chapter_list/ChapterListDialogView.java
package huce.fit.appreadstories.dialog.chapter_list;

import java.util.List;

import huce.fit.appreadstories.dialog.BaseDialogView;
import huce.fit.appreadstories.model.Chapter;
import huce.fit.appreadstories.model.Story;

public interface ChapterListDialogView extends BaseDialogView {
    void setData(Story story);
    void setChapterList(List<Chapter> chapterList);
}

File: app/src/main/java/huce/fit/appreadstories/dialog/chapter_list/ChapterListDialogView.kt
package huce.fit.appreadstories.dialog.chapter_list

import huce.fit.appreadstories.dialog.BaseDialogView
import huce.fit.appreadstories.model.Chapter
import huce.fit.appreadstories.model.Story

interface ChapterListDialogView : BaseDialogView {
    fun setData(story: Story)
    fun setChapterList(chapterList: List<Chapter>)
}
